docs-markdown/src/controllers/alert-controller.ts

```diff
@@ -47,21 +47,20 @@ export function insertAlert() {
             }
             if (editor) {
                 insertContentToEditor(editor, insertAlert.name, formattedText, true);
-
                 if (qpSelection.startsWith("Note")) {
-                    reporter.sendTelemetryEvent("command", { command: telemetryCommand + ".note" });
+                    reporter.sendTelemetryEvent(`${telemetryCommand}.note`, undefined, undefined);
```

Comment:
Do we need to stub out the undefined? Arn't these two optional parameters?

Reply:
Good point, I think they were required in the past but probably not any longer.  I'll check it out now.

Reply:
@bharney-msft Looks much better, good idea.